doc/lightning-listpeers.7.txt

```diff
@@ -0,0 +1,63 @@
+LIGHTNING-LISTPEERS(7)
+=======================
+:doctype: manpage
+
+NAME
+----
+lightning-listpeers - Command for returning data on connected lightning nodes.
+
+SYNOPSIS
+--------
+*listpeers* ['id']
+
+DESCRIPTION
+-----------
+The *listpeers* RPC command returns data on nodes in the Lightning Network
+connected to this node.
+
+Once a connection to another lightning node has been established, using the
+*connect* command, data on the node can be returned using *listpeers* and the
+'id' that was used with the *connect* command.
+
+Supplying 'id' will return only data on a connected lightning node with a
+matching 'id', if one exists.
+
+If no 'id' is supplied, then data on all connected lightning nodes is returned.
+
+RETURN VALUE
+------------
+On success, a "peers" object is returned, containing an array of 0 or more
```

Comment:
Technically it's an object with a `peers` key that is a  list of objects.

Reply:
Happy to change this.  I've checked through and made other references consistent (e.g. using list instead of array)